RetailCoder.VBE/UI/RubberduckUI.resx

```diff
@@ -2052,4 +2052,13 @@ Would you like to import them to Rubberduck?</value>
     <value>Method '{0}' is an implementation of event '{1}.{2}'.  Rename event '{2}'?</value>
     <comment>0: Selected target Identifier; 1: Event Parent; 2: Event name</comment>
   </data>
+  <data name="GeneralSettings_EnableSourceControl" xml:space="preserve">
+    <value>Enable Source Control.  (Restart Rubberduck to take effect.)</value>
```

Comment:
Why two spaces? also I'd prefer the formulation "Requires a restart to take effect."